tests/unit/matrix/test_cubemeasure.py

```diff
@@ -2235,6 +2235,26 @@ def it_knows_its_columns_pruning_base(self):
             np.array([15, 17, 19]),
         )
 
+    def it_knows_its_rows_base(self):
+        unweighted_counts = np.array(
+            [
+                [  # -- row 0 ------------
+                    [1, 2, 3],  # -- selected --
+                    [4, 5, 6],  # -- not --
+                ],
+                [  # -- row 1 ------------
+                    [7, 8, 9],  # -- selected --
+                    [3, 2, 1],  # -- not --
+                    # --------------------
+                ],
+            ]
+        )
+        cube = _MrXCatMatrix(None, None, unweighted_counts, None)
+
+        assert cube.rows_base.tolist() == [12, 48]
```

Comment:
i don't understand this test cause we're using the `_BaseResultMatrix` class that If i well remember is something we should get rid of

Reply:
Yeah, this tests was needed to get coverage back to 100%, I had thought leaving in the legacy code for now and then taking it out in one fell swoop (and maybe with tests) made sense. Do you think I should just take it out? Or put a "no coverage" directive?

Reply:
if this is needed for the 100% coverage means that the `rows_base` measure is not fully consolidated with the new architecture. I would understand what's is missed here.